Lib/test/test_compile.py

```diff
@@ -2605,6 +2608,7 @@ def test_return_inside_async_with_block(self):
             """
         self.check_stack_size(snippet, async_=True)
 
+@unittest.skipIf(_testinternalcapi is None, 'need _testinternalcapi module')
```

Comment:
Might be better to mark this as cpython only, and then fail in cpython if _testinternalcapi is not available.

Reply:
> Might be better to mark this as cpython only, and then fail in cpython if _testinternalcapi is not available.

You're right. It looks like that the test is only appropriate for CPython.